src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpManagementHierarchyController.cs

```diff
@@ -196,12 +193,11 @@ public async Task<IActionResult> EditManagementHierarchy(Guid applicationId, int
         }
 
         [HttpPost]
-        [ModelStatePersist(ModelStatePersist.Store)]
         public async Task<IActionResult> UpdateManagementHierarchyDetails(AddEditManagementHierarchyViewModel model)
         {
             if (!ModelState.IsValid)
             {
-                return RedirectToAction("EditManagementHierarchy", model);
+                return View($"~/Views/Roatp/ManagementHierarchy/EditManagementHierarchy.cshtml", model);
```

Comment:
Just check if the model has not lost any of the parameter set in the GET method.

Reply:
Added unit tests